skimage/feature/blob.py

```diff
@@ -433,7 +433,7 @@ def blob_log(
         Input grayscale image, blobs are assumed to be light on dark
         background (white on black).
     min_sigma : scalar or sequence of scalars, optional
-        the minimum standard deviation for Gaussian kernel. Keep this low to
+        The minimum standard deviation for Gaussian kernel. Keep this low to
         detect smaller blobs. The standard deviations of the Gaussian filter
         are given for each axis as a sequence, or as a single number, in
```

Comment:
```suggestion
        Minimum standard deviation for Gaussian kernel. Keep this value low to
        detect smaller blobs. The standard deviation of the Gaussian kernel
        is given either as a sequence for each axis, or as a single number, in
```
* 1st line: consistent telegraphic style
* 2nd line: singular otherwise we can't have "it is equal ..." on the 3rd line